ui/app/AppLayouts/Wallet/popups/swap/SwapModalAdaptor.qml

```diff
@@ -17,6 +17,22 @@ QObject {
     required property WalletStore.SwapStore swapStore
     required property SwapInputParamsForm swapFormData
 
+    readonly property alias suggestedRoutes: d.suggestedRoutes
+
+    QtObject {
+        id: d
+
+        readonly property string uuid: Utils.uuid()
```

Comment:
this should probably be called each time a new request is made in order to have unique uuids for each request and not made static this way